content/blog/2021/cfp_help.md

```diff
@@ -0,0 +1,45 @@
+Title: How to write a proposal for the talk in PyCon? Our Mentorship team is here to help
+Date: 2021-04-14 10:00
+Author: Dinesh Kumar P (@dinuswt22)
+Category: 2021
+Slug: writing-proposal-pycon-mentorship-team-help
+Summary: Announcing PyCon India Mentorship team's resources, contact links, and weekly calls for first-time speakers as well as veterans.
+
+Do you wish to submit a proposal to PyCon India 2021 but cannot figure out the flow of your talk or how to present it?
+
+Do you have that one lighting topic that can make the Pythonists feel awesome? (Silicon Valley Fans, Assemble!)
+
+Did you solve anything great in Python recently and think it would be great to share it with others?
+
+You have been working in Python for years and interested in presenting your first conference talk?
+
+We know it's not always easy to find the right words when you have something to share. It happens to the best of us. But no need to worry, we’ve all been there. The PyCon India mentorship team is here to help you out with your proposal. Whether you are submitting a proposal at PyCon India for the first time or you are a veteran speaker with years of experience behind you, our mentorship team will guide you through the process and help you wherever required in a number of ways as mentioned below. We want to help you put your best foot forward when it comes to submitting proposals!
+
+### **Recommended Resources from the Content Team**
+
+Prospective speakers of PyCon India can take a look at our curated blog posts on writing better talk and workshop proposals.
+
+* [Tips on writing a “kickass” proposal](https://in.pycon.org/blog/2020/2020-workshop-cfp-announcement.html)
+* [Guidelines on creating video previews](https://in.pycon.org/blog/2020/2020-call-for-proposals-announcement.html)
+
+### **Join us on Zulip for faster answers to any questions**
+
+You can always contact our mentors on Zulip. You can join them on the [PyCon India Zulip Channel](https://pyconindia.zulipchat.com/#) and drop a text in the [pre-cfp topic](https://pyconindia.zulipchat.com/#narrow/stream/245487-wg.2Fmentorship/topic/pre-cfp) of the #Mentorship stream (Streams are available to choose from the sidebar on the left).
```

Comment:
Instead of `#Mentorship stream`, it should be `#wg/mentorship stream`

Reply:
Updated. Please check